docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,81 @@
+[id='proc_updating-booster-deployment-configuration_{context}']
```

Comment:
As per my previous comment, the ID should be `[id='updating-the-deployment-configuration-of-a-booster_{context}']`.

Reply:
Oh, and I forgot that the file name should be `proc_updating-the-deployment-configuration-of-a-booster.adoc`.